DeepProjection/pdbGenerator.py

```diff
@@ -36,36 +36,176 @@ def get_random_pdb(self):
                 pass
 
     
-    def download_all_existing_pdb(self):
-        # Downloads all PDB files from the Protein Data Bank
+    def download_pdb_in_range(self, lower, upper, dir):
+        """
+        Downloads all PDB files from the Protein Data Bank within the
+        range of [lower, upper]
 
+        Parameters
+        ----------
+        lower : str
+            A PDB ID that will be the lower bound of our search.
+            The lower range input will be considered a "wildcard" if the string contains
+            an '*' after a value (e.g. 1A*) or the PDB ID is incomplete (e.g. 1A)
+        
+        upper : str
+            A PDB ID that will be the upper bound of our search.
+            The upper range input will be considered a "wildcard" if the string contains
+            an '*' after a value (e.g. 1A*) or the PDB ID is incomplete (e.g. 1A)
+
+        dir : str
+            The directory to store the PDB files in.
+
+        Corner Cases
```

Comment:
good job thinking about corner cases.
Add these tests to the /tests folder and run pytest and see if they pass